contents/docs/batch-exports/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Batch Exports (beta)
+sidebar: Docs
+showTitle: true
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+---
+
+PostHog Batch Exports gives you a platform to schedule data exports to supported destinations. PostHog Batch Exports is built on [Temporal](https://www.temporal.io/) to enable reliable data exports, ensuring your data reaches your destination.
+
```

Comment:
nit: maybe add some selling points about the change, notably:
  - lower costs thanks to bigger inserts
  - better retry capabilities for resiliency when the destination is temporarily unavailable

No need to go too far, we'll iterate on it.

Reply:
I've added a couple of selling points based on this. But yeah, we do want to iterate 👍